packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -3107,10 +3108,11 @@ export class ProjectService extends BaseService {
             project_uuid: projectUuid,
             user_uuid: user.userUuid,
         };
-        const schema = ProjectService.getWarehouseSchema(credentials);
+        const warehouseSchema =
```

Comment:
This was the main issue, we were assuming all tables were from the same `schema`, (eg: jaffle) but that is not correct if you select a table in a different schema, so I pass schema as a query parameter.